python/pipeline/preprocess.py

```diff
@@ -130,6 +130,21 @@ def estimate_num_components_per_slice(self):
 
             return int(round(num_components))
 
+        def estimate_neuron_size_in_pixels(self):
+            """ Estimates the size of a neuron in the scan (in pixels). Assumes neurons
+             are 15 x 15 microns."""
```

Comment:
At the risk of being pedantic, I think you should refer to the `soma` or `cell body` since most of the neuron is not in the soma.